packages/safe-ds-vscode/src/extension/messages.ts

```diff
@@ -105,6 +125,10 @@ export interface PlaceholderValue {
     name: string;
     type: string;
     value: string;
+    windowed?: boolean;
+    window_begin?: number;
+    window_size?: number;
+    window_max?: number;
```

Comment:
I have a similar question here. Though the combination `windowed: false` and `window_size: 50` doesn't seem to make much sense either way. It seems like in this case, either all fields must be set or none. Thus, I'd propose to add some nesting here:

```ts
{
    // ...
    value: string;
    window?: {
        begin: number;
        size: number;
        max: number;
    }
}
```

For consistency, we should then also use this nested structure for the `PlaceholderQuery`.